python/graphstorm/config/argument.py

```diff
@@ -748,19 +752,72 @@ def eval_fanout(self):
             # By default use -1 as full neighbor
             return [-1] * self.num_layers
 
+    @property
+    def textual_data_path(self):
+        """ distillation textual data path
+        """
+        if hasattr(self, "_textual_data_path"):
+            return self._textual_data_path
+        return None
+
+    @property
+    def lm_name(self):
+        """ HuggingFace Transformer-based model name for distillation
+        """
+        if hasattr(self, "_lm_name"):
+            return self._lm_name
+        return None
+
+    @property
+    def pre_trained_name(self):
+        """ Name of pre-trained HuggingFace model weights for distillation
+        """
+        if hasattr(self, "_pre_trained_name"):
+            return self._pre_trained_name
+        return None
+
+    @property
+    def max_global_step(self):
+        """ Maximum global size for distillation.
+        """
+        # only needed by distillation
+        if "distill" not in self.configuration["gsf"]:
```

Comment:
You do not need to check this as you have a default value and only distillation will access this property.

BTW, you should check whether self._max_global_step > 0.

Reply:
sure, will modify the check